src/libstd/build.rs

```diff
@@ -19,6 +19,7 @@ use std::fs::File;
 
 fn main() {
     let target = env::var("TARGET").expect("TARGET was not set");
+    /*
```

Comment:
I assume the final version will actually delete code, rather than commenting it out?

Reply:
Yes.